default.nix

```diff
@@ -34,6 +35,7 @@ buildGoApplication rec {
   modules = ./gomod2nix.toml;
   pwd = src; # needed to support replace
   subPackages = [ "cmd/cronosd" ];
+  buildFlags = lib.optionalString coverage "-cover";
```

Comment:
this shows warnings in gomod2nix -- probably a todo to fix when it's possible to directly configure it there?

Reply:
yeah, I just opened an issue there: https://github.com/nix-community/gomod2nix/issues/104